internals/cli/masker/writer.go

```diff
@@ -32,7 +32,7 @@ func (m *sequenceMatcher) Read(in byte) int {
 		return 0
 	}
 
-	m.findShift()
+	m.currentIndex -= m.findShift()
```

Comment:
Better! :+1: 

I have one more proposal, how about:

```Go
shift := m.findShift()
if shift > 0 {
    m.currentIndex -= shift
    return m.Read(in)
}
m.currentIndex = 0
return 0
```

I think this makes it clearer when the if statement is true.

Reply:
Shift is always `>0`.

Reply:
Made it a bit simpler though

Reply:
Ahh, that was related to https://github.com/secrethub/secrethub-cli/commit/d56093a0680553b739de6d1da502219bde8fd8e2#r33578665

Reply:
Then we can drop the `m.currentIndex = 0` after the if statement, right? Because when the if statement is not true, shift was `m.currentIndex`, so the new index became 0 after substracting the shift.

Reply:
Yeah. I came to the same conclusion.